docker-compose.nodes.yaml

```diff
@@ -13,6 +13,7 @@ services:
       - /etc/letsencrypt/:/etc/letsencrypt/:ro
   full_node:
     container_name: cere_full_node
+    hostname: ${NODE_NAME}
```

Comment:
Please, add it for:
1.`full_node`, `archive_node` and `validator_node` here
2. `boot_node`,  `add_validation_node_custom`, `full_node` and `archive_node` in the docker-compose.yaml

Reply:
@AndreiNavoichyk Please review additions